### PR TITLE
WSTEAMA-1428 - Update README docs for JumpTo component

### PR DESCRIPTION
**Pull request by louisearchibald**
Resolves JIRA [[1428](https://jira.dev.bbc.co.uk/browse/WSTEAMA-1428)]

Overall changes
======

Updates to README documentation following input from other engineers and our Accessibility Lead.

Code changes
======

- README file updated.

Testing
======
Can be viewed here - https://wsteama-1428-update-readme-docs-for-j--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/docs/components-jumpto--readme&globals=backgrounds.grid:!false

### CODE REVIEW

File: src/app/components/JumpTo/README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-## Description - WORK IN PROGRESS - COMPLETE IN SUBTASK
+## Description
 
-The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific headings within an article. This component renders an `H2`, a list of section titles as `anchor` links, and groups them within a `navigation` landmark. When a link is actioned, the page scrolls down to the relevant section, identified by matching anchor `ids`.
+The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific headings within an article. This component renders an `H2` heading, a list of anchor titles as `anchor` links, and groups them within a `navigation` landmark. When a link is actioned, the page scrolls down to the relevant heading, identified by matching anchor `ids`.
```
Comment (amoore108) [inline src/app/components/JumpTo/README.md]: Don't think it renders an `H2`, its a `strong` element.

Comment (louisearchibald) [reply src/app/components/JumpTo/README.md]: Changed

**Review by amoore108 — APPROVED** [review]
Nice one! 👍

**Review by Louis-Matsika — APPROVED** [review]

File: src/app/components/JumpTo/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
-## Description - WORK IN PROGRESS - COMPLETE IN SUBTASK
+## Description
 
-The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific headings within an article. This component renders an `H2`, a list of section titles as `anchor` links, and groups them within a `navigation` landmark. When a link is actioned, the page scrolls down to the relevant section, identified by matching anchor `ids`.
+The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific headings within an article. This component renders text as a `strong` element, a list of anchor titles as `anchor` links, and groups them within a `navigation` landmark. When a link is actioned, the page scrolls down to the relevant heading, identified by matching anchor `ids`.
```
Comment (greenc05) [inline src/app/components/JumpTo/README.md]: Not sure we need the text I've highlighed in bold below?

The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific headings within an article. This component renders text as a `strong` element, a list of **anchor titles as** `anchor` links, and groups them within a `navigation` landmark. When a link is actioned, the page scrolls down to the relevant heading, identified by matching anchor `ids`.

Comment (louisearchibald) [reply src/app/components/JumpTo/README.md]: Have updated 😄

**Pull request merged**